.github/workflows/firesim-run-tests.yml

```diff
@@ -256,9 +256,22 @@ jobs:
         with:
           test-name: "CIGroupB"
 
+  run-test-bridges:
```

Comment:
For now refer to these as `firesim-lib` bridges. We started talking about renaming these to berkeley-bridges or something more informative, but i need to pull all ofthe non-bridge stuff out of firesim-lib first.